packages/ckeditor5-dev-tests/tests/utils/automated-tests/getkarmaconfig.js

```diff
@@ -9,27 +9,60 @@ const mockery = require( 'mockery' );
 const { expect } = require( 'chai' );
 const sinon = require( 'sinon' );
 const path = require( 'path' );
+const proxyquire = require( 'proxyquire' );
 
 describe( 'getKarmaConfig', () => {
-	let getKarmaConfig, sandbox;
+	let getKarmaConfig, sandbox, stubs;
 	const originalEnv = process.env;
 
 	beforeEach( () => {
 		sandbox = sinon.sandbox.create();
 		sandbox.stub( process, 'cwd' ).returns( 'workspace' );
-
-		// sinon cannot stub non-existing props.
-		process.env = Object.assign( {}, originalEnv, { TRAVIS: false } );
 		sandbox.stub( path, 'join' ).callsFake( ( ...chunks ) => chunks.join( '/' ) );
 		sandbox.stub( path, 'sep' ).value( '/' );
+		// Sinon cannot stub non-existing props.
+		process.env = Object.assign( {}, originalEnv, { TRAVIS: false } );
+
+		stubs = {
+			fs: {
+				writeFileSync: sandbox.stub()
+			},
+			log: {
+				info: sandbox.stub(),
+				warn: sandbox.stub(),
+				error: sandbox.stub(),
+			},
+			glob: {
+				sync: sandbox.stub()
+			},
+			tmp: {
+				fileSync: sandbox.stub().callsFake( () => stubs.tmp._fileSync ),
+				_fileSync: {
+					name: '',
+					removeCallback: sandbox.stub()
+				}
+			}
+		};
 
 		mockery.enable( {
```

Comment:
Does it work with `useCleanCache` too? :D

Reply:
You can check :P

Reply:
It works, I was actually surprised :D